js/test/test.js

```diff
@@ -13,14 +13,18 @@ const fs = require ('fs')
     , { Agent } = require ('https')
     , ccxt = require ('../../ccxt.js') // eslint-disable-line import/order
 
+function dump(...args) {
```

Comment:
we need this in JS too, because we will have to transpile unified `dump` method in test file across langs.